std_msgs/Header header
ChainVehicleState[] data